Makefile

```diff
@@ -14,6 +14,11 @@ help: ## This help.
 # thanks to https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 .DEFAULT_GOAL := help
 
+build_nextcloud: ## Build Nextcloud
+	composer install --no-dev -o && \
```

Comment:
* not sure about `composer install` since we don't do any composer jobs -  lets keep it for now